File: examples/send.cs
using System;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;

class Program
{
    private static async Task AlarmMessage(string message)
    {
        var alarm = "http://t69.me";
        var key = "71b6ffb3-7cff-403e-a2f7-15e3d919f1e1";
        var client = new HttpClient();

        var data = new StringContent($"{{\"key\":\"{key}\",\"message\":\"{message}\"}}", Encoding.UTF8, "application/json");

        try
        {
            var response = await client.PostAsync(alarm, data);

            if (!response.IsSuccessStatusCode)
            {
                Console.WriteLine($"Alarm request failed with status {response.StatusCode}: {response.ReasonPhrase}");
            }
            else
            {
                Console.WriteLine("Alarm sent successfully");
            }
        }
        catch (Exception ex)
        {
            Console.WriteLine($"Alarm error for {key}: {ex.Message}");
        }
    }

    static async Task Main(string[] args)
    {
        await AlarmMessage("😀 Hello World");
    }
}

File: examples/send.js
const axios = require('axios');

async function alarmMessage(message) {
    const alarm = 'http://t69.me';
    const key = '71b6ffb3-7cff-403e-a2f7-15e3d919f1e1';

    try {
        const response = await axios.post(alarm, {
            key: key,
            message: message
        });

        if (response.status !== 200) {
            console.error(`Alarm request failed with status ${response.status}: ${response.data}`);
        } else {
            console.log("Alarm sent successfully");
        }
    } catch (error) {
        console.error(`Alarm error for ${key}: ${error}`);
    }
}

alarmMessage("😀 Hello World");
